src/app/db/tasks.py

```diff
@@ -47,7 +47,7 @@ async def store_in_timeline(
                     }
                     for entry in data
                 ],
-                "form": form,
+                "form": mode,
```

Comment:
should the key also be changed? I think it is renamed in the database model as well!

Reply:
Aaaah, thank you for spotting this. That would have been trouble down the road.